cura/CuraApplication.py

```diff
@@ -1362,3 +1362,7 @@ def _onContextMenuRequested(self, x: float, y: float) -> None:
                     node = node.getParent()
 
                 Selection.add(node)
+
+    @pyqtSlot(result=str)
+    def getCuraVersion(self):
+        return CuraVersion
```

Comment:
This is unnecessary. Take a look at how the version is displayed here: https://github.com/Ultimaker/Cura/blob/master/resources/qml/AboutDialog.qml#L43